integreat_cms/cms/templates/organizations/organization_list.html

```diff
@@ -21,6 +21,7 @@ <h1 class="heading">{% translate "Manage Organizations" %}</h1>
                     <th class="text-sm text-left uppercase py-3 px-2">{% translate "Members" %}</th>
                     <th class="text-sm text-left uppercase py-3 px-2">{% translate "Maintained Pages" %}</th>
                     <th class="text-sm text-left uppercase py-3 px-2">{% translate "Logo" %}</th>
+                    <th class="text-sm text-left uppercase py-3 px-2">{% translate "Website" %}</th>
```

Comment:
```suggestion
                    <th class="text-sm text-left uppercase py-3 pr-2">{% translate "Logo" %}</th>
                    <th class="text-sm text-left uppercase py-3 pr-2">{% translate "Website" %}</th>
```
I'd use the same padding for the table headline like for the table cells